Sources/Datadog/DatadogInternal/DatadogV1CoreProtocol.swift

```diff
@@ -34,6 +34,9 @@ internal protocol DatadogV1CoreProtocol: DatadogCoreProtocol {
 
     /// The SDK context created upon core initialization or `nil` if SDK was not yet initialized.
     var context: DatadogV1Context? { get }
+
+    /// Telemetry monitor for this instance of the SDK or `nil` if not configured.
+    var telemetry: Telemetry? { get }
```

Comment:
I had to do this addition to `DatadogV1CoreProtocol` for simplicity. It is IMO fine as long as it is `V1Core`. For V2 we may want to do it differently - I think we don't yet know how 🙂.

Reply:
I think it should be part of the `DatadogV1Context` instead, no?

Reply:
It could be, although it won't translate to V2 context in any way. Telemetry will be an active feature in V2 (the Feature sends telemetry), not a passive one (the Feature reads telemetry context or sth). Hence, I think whenever we put it, it will be a subject of significant change. WDYT?

Reply:
Yes, not a big deal at this stage!